packages/design-system-react/src/components/BadgeNetwork/BadgeNetwork.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { BadgeNetwork } from './BadgeNetwork';
+
+describe('BadgeNetwork', () => {
+  it('always applies Xs size + border classes for the outer element', () => {
+    render(<BadgeNetwork name="TestNet" src="icon.png" data-testid="badge" />);
+
+    const wrapper = screen.getByTestId('badge');
+    const classList = wrapper.className.split(/\s+/);
+
+    expect(classList).toEqual(
+      expect.arrayContaining([
+        'h-[17px]',
+        'w-[17px]',
```

Comment:
Once we are using the correct sizes for Avatars with border this test will need to be updated we may not necessarily want to test for default props

Reply:
default props is still a test we should check. if a user does not enter a prop and theres a default to it, why dont you think its necessary to check if the default is rendered correctly?

Reply:
and this is the correct size for Avatars with border (16px avatar + 1 px border)